sdk/servicebus/service-bus/src/core/linkEntity.ts

```diff
@@ -471,7 +466,10 @@ export abstract class LinkEntity<LinkT extends Receiver | AwaitableSender | Requ
     this._logger.verbose(
       `${this._logPrefix} Connection is reopening, aborting link initialization.`
     );
-    const err = new MessagingError("Connection is reopening, aborting link initialization.");
+    const err = new ServiceBusError(
+      "Connection is reopening, aborting link initialization.",
+      "GeneralError"
```

Comment:
Can we use `ServiceBusErrorCode.GeneralError` here?

Reply:
ServiceBusErrorCode is a type, not something we can access at runtime (it's a union of strings)